pkg/api/ldap_debug_test.go

```diff
@@ -240,6 +245,7 @@ func TestGetUserFromLDAPApiEndpoint_WithTeamHandler(t *testing.T) {
 		Name:           "John Doe",
 		Email:          "john.doe@example.com",
 		Login:          "johndoe",
+		Groups:         []string{"cn=admins,ou=groups,dc=grafana,dc=org"},
```

Comment:
@gotjosh I didn't quite understand what we were testing here.

Reply:
That the team handler is present - which is a command only available on enterprise mode.  It's a very subtle detail: the `teams` key on the payload is `[]` instead of `null`

Reply:
I could have asserted against _only_ that key but that wouldn't cover changes made to the payload as a whole and I would rather have that coverage.

Reply:
Thanks for clarifying.